avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -1249,7 +1310,30 @@ public void testToConnectUnionRecordConflict() {
     GenericRecord recordTest = new GenericRecordBuilder(avroRecordSchema1).set("test", 12).build();
     avroData.toConnectData(avroSchema, recordTest);
   }
-  
+
+  @Test
+  public void testToConnectFixedUnion() {
+    org.apache.avro.Schema sampleSchema = avroFixed("sample", 4);
+    org.apache.avro.Schema otherSchema = avroFixed("other", 6);
```

Comment:
should any of these other tests have 3 branches in the union, like the one above?  Perhaps it is safest to do that for all of them.

Reply:
Added.